### PR TITLE
es6-continued 

### PR DESCRIPTION
**Pull request by yuriifishchuk**

### CODE REVIEW

**Review by denysdovhan — CHANGES_REQUESTED** [review]
Split this PR into two separate: browser api and es-continued.

**Comment by denysdovhan** [comment]
@komarnitskyi and @denis-zavgorodny your review is needed.

File: homeworks/es6-continued/yuracr9/promisify.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    if (this.status != 200) {
+      return cb(new Error(this.statusText));
+    }
+
+    return cb(null, this.responseText);
+  }
+}
+
+function promisify(func) {
```
Comment (komarnitskyi) [inline homeworks/es6-continued/yuracr9/promisify.js:21]: This promisify fn will accept only one argument. Try to make possible put many arguments in promisified fn and make another PR

Comment (denysdovhan) [reply homeworks/es6-continued/yuracr9/promisify.js:21]: > put many arguments

@komarnitskyi there's no need for that. [Check out the task](https://github.com/kottans/chernivtsi-frontend-2017/tree/master/homeworks/es6-continued#task-3-promisify).

> make another PR

Just push new commits in this PR.
